src/main/scala/com/fulcrumgenomics/illumina/RunInfo.scala

```diff
@@ -54,6 +55,14 @@ case class RunInfo
 ) extends Metric
 
 object RunInfo {
+  /** A set of functions to parse dates in various formats. */
+  val DateParsers: Seq[String => Option[Iso8601Date]] = Seq(
+    d => {if (d.length == 6) Some(new Iso8601Date("20" + d.substring(0,2) + "-" + d.substring(2,4) + "-" + d.substring(4))) else None },
+    d => {if (d.length == 8) Some(new Iso8601Date(d.substring(0,4) + "-" + d.substring(4,6) + "-" + d.substring(6))) else None},
+    d => {try {Some(new Iso8601Date(new SimpleDateFormat("M/d/yyyy h:mm:ss a").parse(d)))} catch { case _: ParseException => None }},
+    d => {try {Some(new Iso8601Date(d))} catch {case _: InvalidDateException => None}}
```

Comment:
Not worth it, but it'd be nicer if this was a method that just returned a `Try[Iso8601Date ]`

Reply:
Yeah, it's a mix - because then I think it would require throwing exceptions in the first two cases, which is not great either.